components/molecules/UserBalance.tsx

```diff
@@ -1,15 +1,79 @@
 import React from "react";
 import {StandardText} from "../typography";
-import {StyleSheet} from "react-native";
+import {StyleSheet, View} from "react-native";
+
 
 export const UserBalance = () => {
-    // TODO -- balance of user
-    const balance = "$8,824.21";
-    return <StandardText style={styles.text}>{balance}</StandardText>;
+
+    const balance = "12453.123";
+
+	const intergerBalance = getIntegerBalance(balance);
+	
+	const decimalBalance = getDecimalBalance(balance);
+
+	return (
+		<View style={styles.container}>
+			<StandardText style={styles.integerBalance}>
+				${intergerBalance}
+			</StandardText>
+			<StandardText style={styles.decimalBalance}>
+				.{decimalBalance}
+			</StandardText>
```

Comment:
Can we try using `SecondaryText` here? so, later, we don't need to use `opacity` or `color` at all